ERCS/erc-7985.md

```diff
@@ -0,0 +1,130 @@
+---
+eip: 7985
+title: Gateway Attributes for Message Control
+description: Gateway attributes for cancellation, timeout, retry, dependencies, and delivery control in cross-chain messaging.
```

Comment:
I don't know what a "gateway attribute" is, and I'm probably representative of the average ERC reader. This is an extremely tiny nit, but if you could choose a different wording in your description that might make it more obvious, I think it would make better use of the limited character space.